src/chapters/chapter7.tex

```diff
@@ -234,7 +234,27 @@ \subsection{Covectors}
 	covectors for $V$, is also a vector space.
 \end{theorem}
 \begin{proof}
-	XXX Finish
+	A common tactic when proving that a particular space is a vector
+    space is to show that it is a subspace of some larger vector space. 
+    This way, most of the properties are inherited and in particular,
+    all we need to show is closure under linear combinations.
+
+    Let $\Omega=\{\textrm{functions from $V$ to $\R$}\}$ equipped with
+    pointwise addition and scalar multiplication. Using an almost identical
+    argument to the one given in Example $7.1$, we see that $\Omega$ is a
+    vector space. We now show that $V^*$ is a subspace of $\Omega$. 
+
+    By definition, $V^*$ is the subset of all linear functions in $\Omega$. 
+    To see that this set is in fact a subspace, take $f,g\in V^*$ and 
+    $\alpha\in \R$. If $h=f+g$, then for any $\vec v,\vec u\in V$ we have 
+    by linearity of $f$ and $g$ that
+    \[
+        h(\alpha\vec v+\vec u)=f(\alpha\vec v+\vec u)+g(\alpha\vec v+\vec u)
+        =\alpha f(\vec v)+\alpha g(\vec v)+f(\vec u)+g(\vec u)
+        =\alpha h(\vec v)+h(\vec u).
+    \]
+    It follows that $h\in V^*$. That is, linear combinations of linear functionals
```

Comment:
No, it doesn't... You never showed that `\alpha h` is still a linear functional.  We also use the word covector in this book.